admin.py

```diff
@@ -0,0 +1,81 @@
+from flask import Flask, render_template, jsonify, request
+from flask_cors import CORS
+from flask_pymongo import PyMongo
+from pymongo import MongoClient
+
+import db
+import json
+import datetime
+from bson import json_util, ObjectId
+from lib import errors
+from lib.utils import clean_json, error, expect_json, success
+
+app = Flask(__name__)
+app.config["SECRET_KEY"] = "ttmadmin"
+CORS(app)
+mdb = MongoClient(db.MONGO_URL).db

```

Comment:
Rather than hosting a separate worker for the admin interface, it's probably better to create a Blueprint:
```suggestion
app = Blueprint('admin', __name__)
```

Reply:
Then in the main application, you can add `app.register_blueprint()` (ex: https://github.com/avrae/avrae-service/blob/master/app.py#L96)

Reply:
To access app globals (e.g. mongo), use `current_app.mdb` (current_app is an import from flask)

Reply:
(For the above, you may have to edit the mongodb declaration line in the main worker from `mdb = MongoClient(...)` to `app.mdb = mdb = MongoClient(...)`)